demo/input_Jacksonville.ini

```diff
@@ -0,0 +1,20 @@
+[paths]
+p3d_fpath = /workdir/Jacksonville/cloud.las
+work_dir = /workdir/Jacksonville
+imagery_dir = /mnt/Jacksonville/WV3
+aoi_config = /configs/Jacksonville_config.json
+
+[aoi]
+name = jacksonville
+
+[params]
+gsd = 0.25
+
+[material]
+model_fpath = /models/Rutgers\ Material\ Segmentation\ Model/model_D4_20.pth.tar
+cuda = True
+
+[roof]
+model_dir = /models/Columbia Geon Segmentation Model
```

Comment:
I think in testing we may want to add back in the backspace for this dir and other configs as well.